ext/standard/tests/skipif_root.inc

```diff
@@ -0,0 +1,16 @@
+<?php
+
+// Skip if being run by root (files are always readable, writeable and executable)
+$filename = @tempnam(__DIR__, 'root_check_');
+if (!file_exists($filename)) {
+    die('WARN Unable to create the "root check" file');
+}
+
+$isRoot = fileowner($filename) == 0;
+
+unlink($filename);
+
+if ($isRoot) {
+    die('SKIP Cannot be run as root');
```

Comment:
Does `SKIP` uppercase work? Same for `WARN`.

Reply:
Yes, it does. The test runner uses [strncasecmp()](https://www.php.net/manual/en/function.strncasecmp.php) to check the presence of these keywords.